### ISSUE TITLE
Erigon error (chain id) after update to CLI 0.14.0

### ISSUE THREAD

**Issue by xtc0r** [lukso-network/tools-lukso-cli#228]
## Type

Erigon does not sync with a chain ID error. I had to delete my mainnet db after updating erigon. Now I'm stuck re-syncing. checkpoint flag is set and ` lukso init ` was done after updating.

## Description

```
[INFO] [12-07|13:21:07.792] logging to file system                   log dir=mainnet-data/execution/logs file prefix=erigon log level=info json=false
[INFO] [12-07|13:21:07.793] Enabling metrics export to prometheus    path=http://127.0.0.1:6060/debug/metrics/prometheus
[INFO] [12-07|13:21:07.793] Build info                               git_branch= git_tag= git_commit=
[INFO] [12-07|13:21:07.794] Maximum peer count                       ETH=100 total=100
[INFO] [12-07|13:21:07.795] starting HTTP APIs                       APIs=erigon,engine
[INFO] [12-07|13:21:07.795] torrent verbosity                        level=WRN
[INFO] [12-07|13:21:09.900] Set global gas cap                       cap=50000000
[INFO] [12-07|13:21:09.981] [Downloader] Runnning with               ipv6-enabled=true ipv4-enabled=true download.rate=16mb upload.rate=4mb
[INFO] [12-07|13:21:09.982] Opening Database                         label=chaindata path=/home/basti/myLUKSOnode/mainnet-data/execution/chaindata
[INFO] [12-07|13:21:09.986] [db] chaindata                           sizeLimit=12TB pageSize=8192
[INFO] [12-07|13:21:09.988] Initialised chain configuration          config="{ChainID: 1, Homestead: 1150000, DAO: 1920000, Tangerine Whistle: 2463000, Spurious Dragon: 2675000, Byzantium: 4370000, Constantinople: 7280000, Petersburg: 7280000, Istanbul: 9069000, Muir Glacier: 9200000, Berlin: 12244000, London: 12965000, Arrow Glacier: 13773000, Gray Glacier: 15050000, Terminal Total Difficulty: 58750000000000000000000, Merge Netsplit: <nil>, Shanghai: 1681338455, Cancun: <nil>, Prague: <nil>, Engine: ethash}" genesis=0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3
[INFO] [12-07|13:21:09.997] Initialising Ethereum protocol           network=42
[INFO] [12-07|13:21:09.997] Disk storage enabled for ethash DAGs     dir=/home/basti/myLUKSOnode/mainnet-data/execution/ethash-dags count=2
[INFO] [12-07|13:21:10.003] Starting private RPC server              on=127.0.0.1:9098
[INFO] [12-07|13:21:10.003] new subscription to logs established 
[INFO] [12-07|13:21:10.004] rpc filters: subscribing to Erigon events 
[INFO] [12-07|13:21:10.004] New txs subscriber joined 
[INFO] [12-07|13:21:10.004] new subscription to newHeaders established 
[INFO] [12-07|13:21:10.005] JsonRpc endpoint opened                  ws=true ws.compression=true grpc=false http.url=127.0.0.1:8545
[INFO] [12-07|13:21:10.007] Reading JWT secret                       path=./configs/shared/secrets/jwt.hex
[INFO] [12-07|13:21:10.007] HTTP endpoint opened for Engine API      url=127.0.0.1:8551 ws=true ws.compression=true
[WARN] [12-07|13:21:10.014] NAT ExternalIP resolution has failed, try to pass a different --nat option err="no UPnP or NAT-PMP router discovered"
[WARN] [12-07|13:21:10.014] [txpool] fromDB: parseTransaction        err="err: rlp parse transaction: invalid chainID, 42 (expected 1), rlp: f90213178477359407830186a094cafe00000000000000000000000000000000cafe8901bc16d674ec800000b901a422895118000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000001207c01aa378b997e7570f4663168da67746aae59645dcb2fb3b0754875adb7805a0000000000000000000000000000000000000000000000000000000000000030a6fb75362884226d073559faf32e891fd52680f54a19318c42ee632b80287ecc8ca74f0f9333563fcc47e433a64488910000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000200100000000000000000000007a53a1bccdf0fa8c65c02d066e532df1c734357a000000000000000000000000000000000000000000000000000000000000006081549ec07aa14b125b9c54ec3d04e99d9639bea02455ceafd08e98f87d6a52fd2ee891794001bf3bd588780e6492a3fc0bd12cf4548b7cfed1a52b5100180066fb001c1bbafbeb3f1922828c720e1839164eed1b38b6854f8bb5659834902cc178a0d375f3cc925f9e4aaff2579c40793e05ec06a25644d1d3df4485015123754a62a0661d493912e36ba6ea4f3ab52724335f408f5c78828b63049758c7998878fd6d"
```
The Invalid chainID error gets repeated several times then log continues like this:

```
[INFO] [12-07|13:21:10.023] [txpool] Started 
[INFO] [12-07|13:21:43.257] [EngineBlockDownloader] Downloading PoS headers... height=unknown hash=0x7d0dfb9c1ad11f70e326c9dcac1a28407121418134bf25aaf6055bbef1b39b89 requestId=0
[INFO] [12-07|13:24:10.001] [p2p] GoodPeers 
[INFO] [12-07|13:24:10.006] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=45.1MB sys=68.7MB
[INFO] [12-07|13:27:09.997] [p2p] GoodPeers 
[INFO] [12-07|13:27:10.005] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=50.3MB sys=72.7MB
[INFO] [12-07|13:30:09.997] [p2p] GoodPeers 
[INFO] [12-07|13:30:10.007] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=35.7MB sys=72.7MB
[INFO] [12-07|13:33:09.997] [p2p] GoodPeers 
[INFO] [12-07|13:33:10.005] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=32.0MB sys=72.7MB
[INFO] [12-07|13:36:09.997] [p2p] GoodPeers 
[INFO] [12-07|13:36:10.017] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=52.6MB sys=72.7MB
[INFO] [12-07|13:39:09.998] [p2p] GoodPeers 
[INFO] [12-07|13:39:10.006] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=51.9MB sys=72.7MB
[INFO] [12-07|13:42:09.999] [p2p] GoodPeers 
[INFO] [12-07|13:42:10.006] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=32.4MB sys=72.7MB
[INFO] [12-07|13:45:09.998] [p2p] GoodPeers 
[INFO] [12-07|13:45:10.005] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=46.3MB sys=72.7MB
[INFO] [12-07|13:48:09.997] [p2p] GoodPeers                          eth67=1
[INFO] [12-07|13:48:10.006] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=35.8MB sys=72.7MB
[WARN] [12-07|13:49:27.378] [downloader] Timeout                     requestId=0 peerID=00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
[WARN] [12-07|13:49:27.387] [EngineBlockDownloader] Header download did not yield success 
[INFO] [12-07|13:49:36.425] [NewPayload] Handling new payload        height=1392574 hash=0x8f34d72d07c03ffbd02adab3b8d8452e58db7d7e593322387bb4ff3b2a05f0c5
[INFO] [12-07|13:49:36.426] [EngineBlockDownloader] Downloading PoS headers... height=unknown hash=0x6e2ac0da2b7e0c406d6954cf0f773ef2fd7d6146b9fe43fe5fe1f944bd06da91 requestId=0
[WARN] [12-07|13:51:06.429] [downloader] Timeout                     requestId=0 peerID=00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
[WARN] [12-07|13:51:06.430] [EngineBlockDownloader] Header download did not yield success 
[INFO] [12-07|13:51:09.998] [p2p] GoodPeers                          eth67=1
[INFO] [12-07|13:51:10.005] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=44.5MB sys=72.7MB
[INFO] [12-07|13:51:12.344] [NewPayload] Handling new payload        height=1392582 hash=0x736097c53ea2d44720b98fbe2fe69ff5d5b2951b240d66986220943d43d61a6f
[INFO] [12-07|13:51:12.344] [EngineBlockDownloader] Downloading PoS headers... height=unknown hash=0xdec46d5c1ea75cb95db8e11318b6ab3dcc59146b0073ddb641905949ac3aa7d1 requestId=0
[WARN] [12-07|13:52:42.348] [downloader] Timeout                     requestId=0 peerID=00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
[WARN] [12-07|13:52:42.349] [EngineBlockDownloader] Header download did not yield success 
[INFO] [12-07|13:52:47.527] [EngineBlockDownloader] Downloading PoS headers... height=unknown hash=0x1d2b60fba1e197e6beffd39fa5b252e3b691247da9fc60ae1e9dc2aa53249f33 requestId=0
[INFO] [12-07|13:54:09.997] [p2p] GoodPeers 
[INFO] [12-07|13:54:10.007] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=42.0MB sys=78.1MB
[WARN] [12-07|13:54:17.535] [downloader] Timeout                     requestId=0 peerID=00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
[WARN] [12-07|13:54:17.536] [EngineBlockDownloader] Header download did not yield success 
[INFO] [12-07|13:54:24.755] [NewPayload] Handling new payload        height=1392597 hash=0x822c335e6f4ddf2238ebb24181f2711f655e82549c3b81170b52c7002b9e30f0
[INFO] [12-07|13:54:24.755] [EngineBlockDownloader] Downloading PoS headers... height=unknown hash=0xb8f0200133d4bc35b2608b6ada656aebcd704cd0c21e19aefc75a6a3754d5443 requestId=0
```

At this point I'm currently stuck.

**Comment by xtc0r** [comment]
I switched back to Lukso cli 0.81 and erigon 2.45.3 and my node is syncing again.

```
[INFO] [12-07|14:51:13.242] logging to file system                   log dir=mainnet-data/execution/logs file prefix=erigon log level=info json=false
[INFO] [12-07|14:51:13.243] Starting metrics server                  addr=http://127.0.0.1:6060/debug/metrics/prometheus
[INFO] [12-07|14:51:13.243] Build info                               git_branch= git_tag= git_commit=
[INFO] [12-07|14:51:13.244] Maximum peer count                       ETH=100 total=100
[INFO] [12-07|14:51:13.244] starting HTTP APIs                       APIs=erigon,engine
[INFO] [12-07|14:51:13.244] torrent verbosity                        level=WRN
[INFO] [12-07|14:51:15.353] Set global gas cap                       cap=50000000
[INFO] [12-07|14:51:15.501] [Downloader] Runnning with               ipv6-enabled=true ipv4-enabled=true download.rate=16mb upload.rate=4mb
[INFO] [12-07|14:51:15.501] Opening Database                         label=chaindata path=/home/basti/myLUKSOnode/mainnet-data/execution/chaindata
mdbx_setup_dxb:15946 filesize mismatch (expect 208896b/51p, have 2147483648b/524288p)
[INFO] [12-07|14:51:15.508] Initialised chain configuration          config="{ChainID: 42, Homestead: 0, DAO: <nil>, Tangerine Whistle: 0, Spurious Dragon: 0, Byzantium: 0, Constantinople: 0, Petersburg: 0, Istanbul: 0, Muir Glacier: <nil>, Berlin: 0, London: 0, Arrow Glacier: <nil>, Gray Glacier: <nil>, Terminal Total Difficulty: 0, Merge Netsplit: <nil>, Shanghai: 1687969198, Cancun: <nil>, Prague: <nil>, Engine: unknown}" genesis=0x5df88817dfb9b00d8ef142370671e8a9bc00c548ab78fbaf205df53db2b24a26
[INFO] [12-07|14:51:15.517] Initialising Ethereum protocol           network=42
[INFO] [12-07|14:51:15.517] Disk storage enabled for ethash DAGs     dir=/home/basti/myLUKSOnode/mainnet-data/execution/ethash-dags count=2
[INFO] [12-07|14:51:15.527] new subscription to logs established 
[INFO] [12-07|14:51:15.527] Starting private RPC server              on=127.0.0.1:9098
[INFO] [12-07|14:51:15.528] rpc filters: subscribing to Erigon events 
[INFO] [12-07|14:51:15.529] new subscription to newHeaders established 
[INFO] [12-07|14:51:15.529] New txs subscriber joined 
[INFO] [12-07|14:51:15.530] Reading JWT secret                       path=./configs/shared/secrets/jwt.hex
[INFO] [12-07|14:51:15.530] HTTP endpoint opened for Engine API      url=127.0.0.1:8551 ws=true ws.compression=true
[INFO] [12-07|14:51:15.531] HTTP endpoint opened                     url=127.0.0.1:8545 ws=true ws.compression=true grpc=false
[INFO] [12-07|14:51:15.537] Started P2P networking                   version=68 self=enode://cef92b00b7eafb8d6f45bc657b09e259c1eb5137c803ed43486acbcb5cd35d35103871960d3ba84c62d882dc3870fea5ab27bc668ae04ae1c2eacf73111247bd@127.0.0.1:0 name=erigon/v2.45.1/linux-arm64/go1.20.4
[INFO] [12-07|14:51:15.539] [txpool] Started 
[INFO] [12-07|14:51:15.540] [2/15 Headers] Waiting for Consensus Layer... 
[INFO] [12-07|14:51:15.542] Started P2P networking                   version=67 self=enode://cef92b00b7eafb8d6f45bc657b09e259c1eb5137c803ed43486acbcb5cd35d35103871960d3ba84c62d882dc3870fea5ab27bc668ae04ae1c2eacf73111247bd@127.0.0.1:0 name=erigon/v2.45.1/linux-arm64/go1.20.4
[INFO] [12-07|14:51:48.956] [2/15 Headers] Downloading PoS headers... height=unknown hash=0x4a22578d17c47907ca368ae5f13eced86a559165ebdc4aec03a0ab28c27335cf requestId=1
[INFO] [12-07|14:52:15.530] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=29.8MB sys=56.3MB
[INFO] [12-07|14:53:15.517] [p2p] GoodPeers 
[INFO] [12-07|14:53:15.530] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=32.0MB sys=56.3MB
[INFO] [12-07|14:54:15.529] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=38.6MB sys=81.4MB
[INFO] [12-07|14:54:45.531] [downloader] Downloaded PoS Headers      now=1376892 blk/sec=294.400
[INFO] [12-07|14:55:15.518] [p2p] GoodPeers                          eth67=1
[INFO] [12-07|14:55:15.530] [downloader] Downloaded PoS Headers      now=1368444 blk/sec=281.600
[INFO] [12-07|14:55:15.530] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=70.0MB sys=117.4MB
[INFO] [12-07|14:55:45.529] [downloader] Downloaded PoS Headers      now=1358652 blk/sec=326.400
```

**Comment by fhildeb** [comment]
Hey, sorry for the inconvenience. Sadly, this issue could not be reproduced from our network team using the default installation version from the LUKSO CLI Version 0.14. (`Erigon v2.55.0`)

Could you please provide the **Erigon version**, **chip**,  and **operating system** that you tested on?

> Did you, by any chance run the `lukso update` command? Updating fetches the latest upstream directly from the client's releases, while the installation only downloads the clients up to the latest version that was tested and verified by the team.

**Comment by xtc0r** [comment]
- erigon 2.55.1 I used `lukso update`for the client
- CPU: rockchip rk 3588
- OS: Ubuntu LTS 22.04 arm

I think a look at the logs is needed. Why is `Erigon 2.55.1` expecting chainID = 1? this also leads to different genesis configuration.


```
Initialised chain configuration          config="{ChainID: 1, Homestead: 1150000, DAO: 1920000, Tangerine Whistle: 2463000, Spurious Dragon: 2675000, Byzantium: 4370000, Constantinople: 7280000, Petersburg: 7280000, Istanbul: 9069000, Muir Glacier: 9200000, Berlin: 12244000, London: 12965000, Arrow Glacier: 13773000, Gray Glacier: 15050000, Terminal Total Difficulty: 58750000000000000000000, Merge Netsplit: <nil>, Shanghai: 1681338455, Cancun: <nil>, Prague: <nil>, Engine: ethash}" genesis=0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3
Initialising Ethereum protocol           network=42
```


Thats from `erigon 2.45.3`

```
Initialised chain configuration          config="{ChainID: 42, Homestead: 0, DAO: <nil>, Tangerine Whistle: 0, Spurious Dragon: 0, Byzantium: 0, Constantinople: 0, Petersburg: 0, Istanbul: 0, Muir Glacier: <nil>, Berlin: 0, London: 0, Arrow Glacier: <nil>, Gray Glacier: <nil>, Terminal Total Difficulty: 0, Merge Netsplit: <nil>, Shanghai: 1687969198, Cancun: <nil>, Prague: <nil>, Engine: unknown}" genesis=0x5df88817dfb9b00d8ef142370671e8a9bc00c548ab78fbaf205df53db2b24a26
Initialising Ethereum protocol           network=42```

**Comment by fhildeb** [comment]
Regarding the chain ID, it might be that the client has to be re-initialized using the`lukso init` command. 

Could you try updating to Erigon 2.55.0 using `lukso install` instead of `lukso update` please? It has some more checks and might catch up with the right LUKSO configuration.

**Comment by xtc0r** [comment]
Like stated in my opening post I used `lukso init`
Where gets the chain ID set? In the `erigon.toml` only? There it was set to 42 when I used 2.55.1.
And what is the difference between `network` and `chainID`? As it seems `network ID` is `42` (correct) but `ChainID` is `1 `(not `42`)

**Comment by fhildeb** [comment]
@Wolmin @mxmar 

Could this have something to do with our network-config file? Maybe this can be adjusted on `Release 0.15`?

**Comment by Hugoo** [comment]
Internal ticket: https://app.clickup.com/t/2645698/DEV-9313

**Comment by Wolmin** [comment]
Hey @xtc0r, sadly we are unable to reproduce this issue. Could you also share the logs from your `lukso start` command? The invalid chain ID may be a cause of some kind of error present during `erigon init` command that is run when `lukso start` is active, in order to correctly prepare the data directory - if you are going to do so, please try with data directory being reset - so we get logs different that "database already present - continuing"

**Comment by xtc0r** [comment]
I gave it a new try today.

1.  Updated Lukso CLI from 0.8.0 to 0.15.0
2. Run `lukso update` to download Erigon v2.55.0 and LH v4.5.0
3. Run `lukso start --validator --transaction-fee-recipient "0x1337" --checkpoint-sync`
4. Check log via `lukso logs execution` and get the following error:
5. 
```
[INFO] [12-19|12:27:19.871] logging to file system                   log dir=mainnet-data/execution/logs file prefix=erigon log level=info json=false
[INFO] [12-19|12:27:19.871] Enabling metrics export to prometheus    path=http://127.0.0.1:6060/debug/metrics/prometheus
[INFO] [12-19|12:27:19.871] Build info                               git_branch= git_tag= git_commit=
[INFO] [12-19|12:27:19.873] Maximum peer count                       ETH=100 total=100
[INFO] [12-19|12:27:19.873] starting HTTP APIs                       APIs=erigon,engine
[INFO] [12-19|12:27:19.874] torrent verbosity                        level=WRN
[INFO] [12-19|12:27:21.984] Set global gas cap                       cap=50000000
[INFO] [12-19|12:27:22.120] [Downloader] Runnning with               ipv6-enabled=true ipv4-enabled=true download.rate=16mb upload.rate=4mb
[INFO] [12-19|12:27:22.121] Opening Database                         label=chaindata path=/home/basti/myLUKSOnode/mainnet-data/execution/chaindata
mdbx_setup_dxb:15897 could not apply preconfigured db-geometry: err -30417
[EROR] [12-19|12:27:22.125] Erigon startup                           err="mdbx_env_open: MDBX_TOO_LARGE: Database is too large for current system, e.g. could NOT be mapped into RAM, label: chaindata, trace: [kv_mdbx.go:295 node.go:365 node.go:368 backend.go:239 node.go:124 main.go:65 make_app.go:52 command.go:274 app.go:332 app.go:309 main.go:34 proc.go:250 asm_arm64.s:1172]"
mdbx_env_open: MDBX_TOO_LARGE: Database is too large for current system, e.g. could NOT be mapped into RAM, label: chaindata, trace: [kv_mdbx.go:295 node.go:365 node.go:368 backend.go:239 node.go:124 main.go:65 make_app.go:52 command.go:274 app.go:332 app.go:309 main.go:34 proc.go:250 asm_arm64.s:1172] 
```

6. Fixed the error by adding `"db.size.limit"="8TB"` to `erigon.toml`. I told @fhildeb in discord already about this breaking change. See here: [erigon 2.55.0 release notes](https://github.com/ledgerwatch/erigon/releases/tag/v2.55.0)
7. Restarted erigon without any issues.
8. Manually downloaded erigon `2.55.1`and repleaced the bin in `/usr/local/bin` because this ticket was opened when I used erigon`2.55.1`as `lukso update` did download this version at this point in time. Fixed now with lukso cli `0.15.0` 
9. Also erigon `2.55.1 ` starts fine. See log below. `ChainID` and `networkID` are both `42`

```
[INFO] [12-19|12:46:51.818] [db] chaindata                           sizeLimit=8TB pageSize=4096
[INFO] [12-19|12:46:51.823] Initialised chain configuration          config="{ChainID: 42, Homestead: 0, DAO: <nil>, Tangerine Whistle: 0, Spurious Dragon: 0, Byzantium: 0, Constantinople: 0, Petersburg: 0, Istanbul: 0, Muir Glacier: <nil>, Berlin: 0, London: 0, Arrow Glacier: <nil>, Gray Glacier: <nil>, Terminal Total Difficulty: 0, Merge Netsplit: <nil>, Shanghai: 1687969198, Cancun: <nil>, Prague: <nil>, Engine: unknown}" genesis=0x5df88817dfb9b00d8ef142370671e8a9bc00c548ab78fbaf205df53db2b24a26
[INFO] [12-19|12:46:51.831] Initialising Ethereum protocol           network=42
[INFO] [12-19|12:46:51.831] Disk storage enabled for ethash DAGs     dir=/home/basti/myLUKSOnode/mainnet-data/execution/ethash-dags count=2
[INFO] [12-19|12:46:52.643] Starting private RPC server              on=127.0.0.1:9098
[INFO] [12-19|12:46:52.643] new subscription to logs established 
[INFO] [12-19|12:46:52.643] rpc filters: subscribing to Erigon events 
[INFO] [12-19|12:46:52.644] New txs subscriber joined 
[INFO] [12-19|12:46:52.644] new subscription to newHeaders established 
[INFO] [12-19|12:46:52.646] JsonRpc endpoint opened                  ws=true ws.compression=true grpc=false http.url=127.0.0.1:8545
[INFO] [12-19|12:46:52.647] Reading JWT secret                       path=./configs/shared/secrets/jwt.hex
[INFO] [12-19|12:46:52.647] HTTP endpoint opened for Engine API      url=127.0.0.1:8551 ws=true ws.compression=true
[WARN] [12-19|12:46:52.649] NAT ExternalIP resolution has failed, try to pass a different --nat option err="no UPnP or NAT-PMP router discovered"
[WARN] [12-19|12:46:52.652] NAT ExternalIP resolution has failed, try to pass a different --nat option err="no UPnP or NAT-PMP router discovered"
[INFO] [12-19|12:46:52.652] Started P2P networking                   version=68 self=enode://cef92b00b7eafb8d6f45bc657b09e259c1eb5137c803ed43486acbcb5cd35d35103871960d3ba84c62d882dc3870fea5ab27bc668ae04ae1c2eacf73111247bd@127.0.0.1:30313 name=erigon/v2.55.1/linux-arm64/go1.20.7
[INFO] [12-19|12:46:52.654] [txpool] Started 
[INFO] [12-19|12:46:52.658] Started P2P networking                   version=67 self=enode://cef92b00b7eafb8d6f45bc657b09e259c1eb5137c803ed43486acbcb5cd35d35103871960d3ba84c62d882dc3870fea5ab27bc668ae04ae1c2eacf73111247bd@127.0.0.1:30314 name=erigon/v2.55.1/linux-arm64/go1.20.7
```

**Conclusion:**
Issue can't be reproduced with `lukso cli 0.15.0` using erigon `2.55.0` nor `2.55.1` but `erigon.toml` needs to be updated to add `"db.size.limit"="8TB"`
You can close the issue. I don't do it to keep you aware of the needed `erigon.toml` change.

**Comment by Wolmin** [comment]
> **Conclusion:** Issue can't be reproduced with `lukso cli 0.15.0` using erigon `2.55.0` nor `2.55.1` but `erigon.toml` needs to be updated to add `"db.size.limit"="8TB"` You can close the issue. I don't do it to keep you aware of the needed `erigon.toml` change.

Thanks for pointing this out - we have created a PR in network config repository so we can avoid this being an issue for other people running their node 👍 - https://github.com/lukso-network/network-configs/pull/129 - since this PR merging process is another thing, I'll close this issue
